sharpy/rom/balanced.py

```diff
@@ -103,7 +124,10 @@ def run(self, ss):
                                     gv=S,
                                     convergence=self.settings['convergence'],
                                     method=self.settings['reduction_method'])
-
+            if librom.check_stability(ssrom.A, dt=True):
+                cout.cout_wrap('ROM by direct balancing is stable')
```

Comment:
These things should definitely be quiet (or wrapped in an `if self.settings['print_info']` for the good outcome (stable ROM). I understand if the output is forced when it is unstable.

Reply:
Included in the latest commits